Resources/config/user_context.xml

```diff
@@ -10,6 +10,7 @@
         <parameter key="fos_http_cache.user_context.role_provider.class">FOS\HttpCacheBundle\UserContext\RoleProvider</parameter>
         <parameter key="fos_http_cache.user_context.request_matcher.class">FOS\HttpCacheBundle\UserContext\RequestMatcher</parameter>
         <parameter key="fos_http_cache.user_context.logout_handler.class">FOS\HttpCacheBundle\Security\Http\Logout\ContextInvalidationLogoutHandler</parameter>
+        <parameter key="fos_http_cache.user_context.anonymous_request_matcher.class">FOS\HttpCacheBundle\UserContext\AnonymousRequestMatcher</parameter>
```

Comment:
parameters for class names is not best practice anymore. lets just put the class name into the service definition directly. we will do that for everything, if not already done with 2.0